src/insurancesCalculator.js

```diff
@@ -1,72 +1,29 @@
 const Big = require('big.js');
-
-const REGIONS = [1, 2, 3, 4];
+const findPolicyAndRegion = require('./findPolicyAndRegion');
 const BIG_20 = new Big(20);
-/**
- * If there are new policy changes, please add that on top!
- */
-const POLICY_UPDATES = [
-    {
-        startDate: new Date("2023-07-01"),
-        regionMinWages:
-            [
-                { region: 1, minWage: 4_680_000 },
-                { region: 2, minWage: 4_160_000 },
-                { region: 3, minWage: 3_640_000 },
-                { region: 4, minWage: 3_250_000 }
-            ],
-        baseSalary: new Big(1_800_000)
-    },
-    {
-        startDate: new Date("2022-07-01"),
-        regionMinWages:
-            [
-                { region: 1, minWage: 4_420_000 },
-                { region: 2, minWage: 3_920_000 },
-                { region: 3, minWage: 3_430_000 },
-                { region: 4, minWage: 3_070_000 }
-            ],
-        baseSalary: new Big(1_490_000)
-    },
-];
 
 /**
- * 
- * @param {Number} gross 
- * @param {1|2|3|4} region 
- * @param {Date} date 
+ *
+ * @param {Number} gross
+ * @param {1|2|3|4} region
+ * @param {Date} date
  * @returns {Promise<{total: number, insurances: { name: string, amount: number }[]}>}
  */
 
 module.exports = function insurancesCalculator(gross, region = 1, date = new Date()) {
     return new Promise(function (resolve, reject) {
 
         const grossBig = new Big(gross);
+        const infoRegion = findPolicyAndRegion(region, date);
+        const minWage = infoRegion.minWage;
```

Comment:
we dont really need to declare `minWage` or `baseSalary` variables here.

we can access it from regionInfo object.